Ray/Approx/Series.lean

```diff
@@ -47,6 +47,13 @@ lemma approx_taylor_sum' (c : Array (Interval)) (c' : ℕ → ℝ) (x p t : Inte
   · rw [taylor_sum']
     simp only [Finset.sum_range_succ', pow_zero, mul_one, add_zero, add_comm (Finset.sum _ _),
       ←add_assoc, pow_succ, ←mul_assoc _ x', mul_assoc _ _ x', add_right_comm _ _ (1:ℕ)]
+    -- TODO: this conv block is just to massage the goal to match an old version,
+    -- this could probably be consolidated with the `simp only` above.
+    conv =>
```

Comment:
TODO: this needs extra review

Reply:
I took out the TODO comment, but this might still be worth @girving taking a second look.

Reply:
I'm fine with it as is.